vendors/pc/boards/windows/aws_demos/application_code/runtimeTimer.c

```diff
@@ -0,0 +1,70 @@
+#include "runtimeTimer.h"
+
+#include "FreeRTOS.h"
+#include "FreeRTOSIPConfig.h"
+#include "task.h"
+#include "timers.h"
+
+#define MS_PER_MINUTE (SECONDS_PER_MINUTE * MS_PER_SECOND)
+#define SECONDS_PER_MINUTE (60)
+#define MS_PER_SECOND (1000)
```

Comment:
These values are not in the header file because they are only used for calculations in this file.